src/server/system_services/master_key_manager.js

```diff
@@ -244,6 +289,12 @@ class MasterKeysManager {
         return db_m_key.disabled === true;
     }
 
+    is_r_key_disabled(id) {
```

Comment:
it seems like this can be merged to is_m_key_disabled() since it's doing the same thing, right?